[2021-06-28 15:20:02,043] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: summary_db.ambassador_channel_partner_comission 2021-06-26T23:45:00+00:00 [queued]>
[2021-06-28 15:20:02,523] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: summary_db.ambassador_channel_partner_comission 2021-06-26T23:45:00+00:00 [queued]>
[2021-06-28 15:20:02,523] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-28 15:20:02,524] {taskinstance.py:1068} INFO - Starting attempt 1 of 4
[2021-06-28 15:20:02,524] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-28 15:20:02,655] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): ambassador_channel_partner_comission> on 2021-06-26T23:45:00+00:00
[2021-06-28 15:20:02,755] {standard_task_runner.py:52} INFO - Started process 307 to run task
[2021-06-28 15:20:02,857] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'summary_db', 'ambassador_channel_partner_comission', '2021-06-26T23:45:00+00:00', '--job-id', '281', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/analytics_summary.py', '--cfg-path', '/tmp/tmp_cynssz2', '--error-file', '/tmp/tmp9_2bdmq_']
[2021-06-28 15:20:02,873] {standard_task_runner.py:77} INFO - Job 281: Subtask ambassador_channel_partner_comission
[2021-06-28 15:20:03,344] {logging_mixin.py:104} INFO - Running <TaskInstance: summary_db.ambassador_channel_partner_comission 2021-06-26T23:45:00+00:00 [running]> on host 32c9461fc328
[2021-06-28 15:20:03,737] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=summary_db
AIRFLOW_CTX_TASK_ID=ambassador_channel_partner_comission
AIRFLOW_CTX_EXECUTION_DATE=2021-06-26T23:45:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-06-26T23:45:00+00:00
[2021-06-28 15:20:03,738] {logging_mixin.py:104} INFO - ------------------------------------Inside Function-----------------------------
[2021-06-28 15:20:05,556] {logging_mixin.py:104} INFO - -------------------------------Data read into dataframe---------------------------
[2021-06-28 15:20:08,242] {logging_mixin.py:104} INFO - ----------------------------------Data written into summary DB----------------------
[2021-06-28 15:20:08,245] {python.py:151} INFO - Done. Returned value was: None
[2021-06-28 15:20:08,316] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=summary_db, task_id=ambassador_channel_partner_comission, execution_date=20210626T234500, start_date=20210628T152002, end_date=20210628T152008
[2021-06-28 15:20:08,939] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-06-28 15:20:09,144] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-06-28 15:38:19,985] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: summary_db.ambassador_channel_partner_comission 2021-06-26T23:45:00+00:00 [queued]>
[2021-06-28 15:38:20,044] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: summary_db.ambassador_channel_partner_comission 2021-06-26T23:45:00+00:00 [queued]>
[2021-06-28 15:38:20,044] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-28 15:38:20,044] {taskinstance.py:1068} INFO - Starting attempt 1 of 4
[2021-06-28 15:38:20,045] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-28 15:38:20,067] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): ambassador_channel_partner_comission> on 2021-06-26T23:45:00+00:00
[2021-06-28 15:38:20,085] {standard_task_runner.py:52} INFO - Started process 292 to run task
[2021-06-28 15:38:20,114] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'summary_db', 'ambassador_channel_partner_comission', '2021-06-26T23:45:00+00:00', '--job-id', '300', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/analytics_summary.py', '--cfg-path', '/tmp/tmpdz250noh', '--error-file', '/tmp/tmpdy9o6pgp']
[2021-06-28 15:38:20,114] {standard_task_runner.py:77} INFO - Job 300: Subtask ambassador_channel_partner_comission
[2021-06-28 15:38:20,276] {logging_mixin.py:104} INFO - Running <TaskInstance: summary_db.ambassador_channel_partner_comission 2021-06-26T23:45:00+00:00 [running]> on host 32c9461fc328
[2021-06-28 15:38:20,396] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=summary_db
AIRFLOW_CTX_TASK_ID=ambassador_channel_partner_comission
AIRFLOW_CTX_EXECUTION_DATE=2021-06-26T23:45:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-06-26T23:45:00+00:00
[2021-06-28 15:38:20,396] {logging_mixin.py:104} INFO - ------------------------------------Inside Function-----------------------------
[2021-06-28 15:38:22,073] {logging_mixin.py:104} INFO - -------------------------------Data read into dataframe---------------------------
[2021-06-28 15:38:23,860] {logging_mixin.py:104} INFO - ----------------------------------Data written into summary DB----------------------
[2021-06-28 15:38:23,867] {python.py:151} INFO - Done. Returned value was: None
[2021-06-28 15:38:23,913] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=summary_db, task_id=ambassador_channel_partner_comission, execution_date=20210626T234500, start_date=20210628T153819, end_date=20210628T153823
[2021-06-28 15:38:23,988] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-06-28 15:38:24,018] {local_task_job.py:151} INFO - Task exited with return code 0
